src/PhpSpec/Matcher/ThrowMatcher.php

```diff
@@ -121,11 +121,25 @@ public function verifyPositive($callable, array $arguments, $exception = null)
         }
 
         if (!$exceptionThrown instanceof $exception) {
+            $format = 'Expected exception of class %s, but got %s.';
+
+            // Show full error message if the thrown exception is an instance of Error.
+            // See https://github.com/phpspec/phpspec/issues/908
+            if (class_exists('\Error') && $exceptionThrown instanceof \Error) {
```

Comment:
Thanks @stof. I've now removed it (https://github.com/phpspec/phpspec/pull/915/commits/d4485909b120ce3bcde383e3c4c6ef09d13816d1)